src/views/Nft/market/utils/getIpfsLinkWithProvider.ts

```diff
@@ -0,0 +1,12 @@
+const BASE_IPFS_LINK = 'https://cloudflare-ipfs.com/ipfs/'
+
+const getIpfsLinkWithProvider = (ipfsJson: string): string | null => {
```

Comment:
There's an helper called `getTokenUrl` which is doing the same already, please have a look at it